tyk-stack/values.yaml

```diff
@@ -939,6 +939,9 @@ tyk-dashboard:
               pathType: ImplementationSpecific
       tls: []
 
+  tib:
+    enabled: false
+
```

Comment:
if we have a TIB component chart in the future, what do you think about having a `tib` field along with separate TIB   field, for instance `tyk-dashboard.tib` and `tyk-tib`? Would it be confusing?